src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$color-accent-blue: rgba(0, 172, 220, 1);
+$color-main-accent: #060b35;
```

Comment:
Consider using more descriptive variable names for colors, such as $main-text-color or $primary-text-color.